### ISSUE TITLE
3D Source

### ISSUE THREAD

**Issue by cgrobb** [microsoft/fluentui-emoji#14]
Thanks for opening this work up!

Was surprised to see that the "3D" data is not in source form, but rendered form. That really limits what kinds of remixing and (machine) learning can be done. What plans are there to provide this?

**Comment by cnkurzke** [comment]
I agree with @cgrobb - having actual 3D sources would make this MUCH more useful.

It would allow us to create actual modifications to the 3D emoji, which is nearly impossible in the PNG format.
It would also allow other uses, like conversion to STL for 3D Printers.

For example a great data format would be the X3D file format. This format is widely used in the 3D modeling community.

The standard:   https://www.web3d.org/
Or also: https://doc.x3dom.org/gettingStarted/background/index.html

**Comment by geuis** [comment]
Also want to add to what others are saying about the flat png format for
3D. It isn't 3D. 

If these were at least height/bump/displacement or even normal maps, we could use them as 3D assets. As-is, these aren't usable as such.

**Comment by theuserbl** [comment]
And where are SVG versions of it?
They are only as png.

**Comment by justsomederpystuff** [comment]
> And where are SVG versions of it? They are only as png.

In the Color directory for each emoji, would probably be good to explain the directory structure in the readme.

**Comment by jasoncuster** [comment]
Hey gang! Thanks for the feedback - we are discussing this. If we did, we'd really want to set expectations [as explained here](https://www.linkedin.com/pulse/making-fluent-emoji-nando-costa/) the emoji aren't fully 3D, they are created in 3D space to look great as a flat object.

Also thanks @justsomederpystuff for the suggestion on the clear file types in the readme. We have it in the Figma files, but it's a great idea to include here as well.

**Comment by cnkurzke** [comment]
Thanks Jason for being open to the suggestions.  Let us know if you need help, for example converting the files to/from different file formats.
e.g. see https://blog.photopea.com/photopea-4-6-open-figma-files.html

There are some Open Source ways to convert Figma files into other formats, but the best resullts would likely be to convert at export time.

**Comment by NayamAmarshe** [comment]
Would be great if we could at least get the 3D ones in 1024px or a guide to convert them ourselves.

**Comment by justsomederpystuff** [comment]
> Would be great if we could at least get the 3D ones in 1024px or a guide to convert them ourselves. We wouldn't need source if we could get them in high-res.

@NayamAmarshe You can use svg files from inside of the `Color` directories for each emoji to render variants for any size you want.

**Comment by NayamAmarshe** [comment]
> > Would be great if we could at least get the 3D ones in 1024px or a guide to convert them ourselves. We wouldn't need source if we could get them in high-res.
> 
> @NayamAmarshe You can use svg files from inside of the `Color` directories for each emoji to render variants for any size you want.

But they're different from the 3D ones. Lacking in shadow and depth.

**Comment by karl-police** [comment]
so where is what now? 😕

**Comment by superusercode** [comment]
Google has provided obj files for their own equivalent emoji https://www.theverge.com/design/967606/google-open-source-3d-emoji it would be nice for us to have parity
